Makefile

```diff
@@ -46,3 +46,25 @@ yq:
 pint_lint:
 	echo "Linting Prometheus rules..."
 	./pint --no-color lint ${EXTRACTED_RULE_FILE}
+
+yamllint:
+	echo "Installing yamllint..."
+	python3 -m pip install pipenv
+	python3 -m pipenv install --dev
+
+.PHONY: yaml_lint
+yaml_lint:
+	echo "Linting YAML files..."
+	python3 -m pipenv run yamllint . && echo "yaml_lint SUCCESS"
+
+.PHONY: install_pipenv
+install_pipenv:
+	python3 -m pip install pipenv
+
+.PHONY: sync_pipenv
+sync_pipenv:
+	python3 -m pipenv sync --dev
+
+.PHONY: lint_yamls
+lint_yamls:
+	python3 -m pipenv run yamllint . && echo "lint_yamls: SUCCESS"
```

Comment:
Did you verify that this can also fail? i.e. temporarily add a yaml that should fail yamllint and verify that ci fails in the way it should.

Reply:
Verified now, it fails in the CI too.